src/singularity.ts

```diff
@@ -466,11 +470,12 @@ replication.command('start')
 replication.command('status')
   .description('Check the status of a deal replication request')
   .argument('<id>', 'A unique id of the dataset')
+  .option('-c, --count <count>', 'Whether to count deal states (performance impacting). true|false.', 'false')
```

Comment:
why is this performance impacting? Does database haave the correct index for this query?

Reply:
We should remove this option and always return the count.